extensions/sql-database-projects/src/projectProvider/projectProvider.ts

```diff
@@ -39,9 +44,22 @@ export class SqlDatabaseProjectProvider implements dataworkspace.IProjectProvide
 	 */
 	get supportedProjectTypes(): dataworkspace.IProjectType[] {
 		return [{
+			id: sqlDatabaseProjectTypeId,
 			projectFileExtension: sqlprojExtension.replace(/\./g, ''),
 			displayName: projectTypeDisplayName,
+			description: projectTypeDescription,
 			icon: IconPathHelper.databaseProject
 		}];
 	}
+
+	/**
+	 * Create a project
+	 * @param name name of the project
+	 * @param location the parent directory
+	 * @returns Uri of the newly created project file
+	 */
+	async createProject(name: string, location: vscode.Uri): Promise<vscode.Uri> {
+		const projectFile = await this.projectController.createNewProject(name, location, true);
```

Comment:
the MainController.createNewProject() should be removed since it isn't being used anymore, but I can clean it up later